Backend/dev/migrations/dynamic-offer-driver-app/0281-add-update-kapture-auth.sql

```diff
@@ -0,0 +1,9 @@
+UPDATE atlas_driver_offer_bpp.merchant_service_config
```

Comment:
We don't need to update service_config as Kapture configs are already there. Lets remove this migration.